client/src/components/Map/Map.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import GoogleMapReact from 'google-map-react';
+ 
+const AnyReactComponent = ({ text }) => <div>{text}</div>;
+ 
+class Map extends Component {
+  static defaultProps = {
+    center: {
+      lat: 49.895138,
+      lng: -97.138374
+    },
+    zoom: 16
+  };
+ 
+  render() {
+    return (
+      // Important! Always set the container height explicitly
+      <div style={{ height: '100vh', width: '100%' }}>
+        <GoogleMapReact
+          bootstrapURLKeys={{ key: "AIzaSyAQFPAvbRpkC_iU7Ah2fkNDEBbCxcMgG8w" }}
```

Comment:
Maybe we should keep API keys in a separate file in case we change them in the future

Reply:
true good idea ill move it